### ISSUE TITLE
Logstash: Handle JVM heap size with a variable

### ISSUE THREAD

**Issue by frankhetterich** [NETWAYS/ansible-collection-elasticstack#229]
At the moment there is no handling for the JVM heap size parameters (Xms/Xmx) for Logstash installations. The jvm.options file is completely untouched and Logstash is using only 1G of memory. 

There should be the same function as for Elasticsearch to use at least half of the available machine memory or configure the Heap size with a variable.

**Comment by widhalmt** [comment]
Good point! Logstash usually doesn't need as much memory as Elasticsearch does. And since some setups run Elasticsearch and Logstash on the same hosts, automatic calculation might not work. But using a variable is definitely a good idea.
